docs/users_guide/landice/test_groups/humboldt.rst

```diff
@@ -28,6 +28,25 @@ The test group uses the following default config options:
     # to cull based on distance from margin.
     cullDistance = 5.0
 
+    # mesh density parameters
+    # minimum cell spacing (meters)
+    minSpac = 1.e3
+    # maximum cell spacing (meters)
+    maxSpac = 1.e4
+    # log10 of max speed (m/yr) for cell spacing
+    highLogSpeed = 2.5
+    # log10 of min speed (m/yr) for cell spacing
+    lowLogSpeed = 0.75
+    # distance at which cell spacing = maxSpac (meters)
+    highDist = 1.e5
+    # distance within which cell spacing = minSpac (meters)
+    lowDist = 1.e4
+    
+    # mesh density functions
+    useSpeed = True
+    useDistToGroundingLine = False
+    useDistToEdge = True
```

Comment:
Config options end up in all lowercase after they get parsed and written out again as a new file.  That makes camel case particularly hard to deal with.  Could you convert these to be lowercase with underscores?  It would be clearer if you convert the corresponding variables in the function to the same naming convention (which is also what python prefers) but I don't feel as strongly about that.

Reply:
Make sure the docstring gets updated with the corresponding changes.